.gitignore

```diff
@@ -110,3 +110,5 @@ lib/matplotlib/backends/web_backend/node_modules/
 lib/matplotlib/backends/web_backend/package-lock.json
 
 LICENSE/LICENSE_QHULL
+bin/
+lib/python3.8/site-packages/
```

Comment:
I'm a bit iffy on adding these to `.gitignore` at all, as they are super specific go an individual workflow... but if they are included, they certainly should not go in the section labeled "Vendored dependencies"

I think including your venv at repo root is an anti-pattern, especially since the `lib` folder is used in this repository, which makes it even more confusing.